test/dashboard.go

```diff
@@ -0,0 +1,147 @@
+package test
+
+import (
+	"net/http"	
+	"bytes"
+	"net/http/cookiejar"
+	"fmt"	
+	"io/ioutil"
+	"encoding/json"
+)
+
+var sessionCookie []*http.Cookie
+var jar *cookiejar.Jar
+var runId string
+var postDataClient  *http.Client
+
+
+func loginDashboard() (*http.Client) {
+		
+	sessionCookie  = nil			
+	jar, _ := cookiejar.New(nil)
+	
+	client := &http.Client{
+		Jar: jar,
+	}
+	
+	var jsonStr = []byte(`{"username":"tester","password":"nimbledcstester"}`)	
+	req, _ := http.NewRequest("POST", "http://10.18.223.131:4000/apilogin", bytes.NewBuffer(jsonStr))
+	
+	req.Header.Add("Content-Type", "application/json")
+	fmt.Println(req)
```

Comment:
Does this print credentials?  That should be avoided.

Reply:
removed println